calico_containers/tests/st/test_base.py

```diff
@@ -13,55 +14,19 @@ class TestBase(TestCase):
     """
     def setUp(self):
         """
-        Clean up host containers before every test.
+        Clean up before every test.
         """
-        # containers = docker.ps("-qa").split()
-        # for container in containers:
-        #     delete_container(container)
-
         self.ip = get_ip()
-        self.start_etcd()
+        # Assert that /calico doesn't exist
```

Comment:
Comment is incorrect.  "Delete /calico if it exists.  This ensures each test has an empty data store at start of day."